mapengine/worldwind/README

```diff
@@ -2,4 +2,4 @@ This repository build and publishes two artifacts:
 
 1. emp3-android-worldwind.aar - Applications can compile with this aar to use the NASA WorldWind Map engine.
 
-2. emp3-android-worldwind.apk - EMP3 SDK class loads the NASE WorldWind map engine from this Map. This is the preferred usage.
+2. emp3-android-worldwind.apk - EMP3 SDK class loads the NASA WorldWind map engine from this Map. This is the preferred usage.
```

Comment:
Would you mind adding a link to the WorldWind GitHub project?

Reply:
Assuming this is WorldwindAndroid? https://github.com/NASAWorldWind/WorldWindAndroid

Reply:
Yes